models/src/it/java/io/sphere/sdk/reviews/ReviewIntegrationTest.java

```diff
@@ -0,0 +1,392 @@
+    // { authorName, title, text, score }
+    private static final Object[][] REVIEWS = {
+            {"First author","First review","This is my first comment",0.3f},
+            {"Second author","Second review","This is my second comment",0.4f},
+            {"Third author","Third review","This is my third comment",0.5f},
+            {"Fourth author","Fourth review","This is my fourth comment",0.6f},
+            {"Fifth author","Fifth review","This is my fifth comment",0.7f}
+    };
+
+    @Test
+    public void testCreate() throws Exception {
+        withProduct(client(), product ->
+            withCustomer(client(), customer -> {
+                Instant start = Instant.now().minusSeconds(SERVER_TIME_TOLERANCE_SEC); // time tolerance
+                Review review = client().execute(ReviewCreateCommand.of(ReviewDraft.of(product.getId(), customer.getId())
+                        .withAuthorName((String) REVIEWS[0][0]).withTitle((String) REVIEWS[0][1])
+                        .withText((String) REVIEWS[0][2]).withScore((Float) REVIEWS[0][3])));
+                Instant afterCreation = Instant.now().plusSeconds(SERVER_TIME_TOLERANCE_SEC); // time tolerance
+                assertEquals(product.getId(), review.getProductId());
+                assertEquals(customer.getId(), review.getCustomerId());
+                assertEquals((String) REVIEWS[0][0], review.getAuthorName().get());
+                assertEquals((String) REVIEWS[0][1], review.getTitle().get());
+                assertEquals((String) REVIEWS[0][2], review.getText().get());
+                assertEquals((Float) REVIEWS[0][3], review.getScore().get());
+                assertNotNull(review.getId());
+                assertEquals(1, review.getVersion());
+                Instant timestamp = review.getCreatedAt();
+                if (!timestamp.isAfter(start) || !timestamp.isBefore(afterCreation)) {
+                    fail(String.format("Invalid creation time %s when expected between %s and %s",
+                            DateTimeFormatter.ISO_INSTANT.format(timestamp),
+                            DateTimeFormatter.ISO_INSTANT.format(start),
+                            DateTimeFormatter.ISO_INSTANT.format(afterCreation)));
+                }
+                timestamp = review.getLastModifiedAt();
+                if (!timestamp.isAfter(start) || !timestamp.isBefore(afterCreation)) {
+                    fail(String.format("Invalid last modified time %s when expected between %s and %s",
+                            DateTimeFormatter.ISO_INSTANT.format(timestamp),
+                            DateTimeFormatter.ISO_INSTANT.format(start),
+                            DateTimeFormatter.ISO_INSTANT.format(afterCreation)));
+                }
+            })
+        );
+
+    }
+
+    @Test(expected = io.sphere.sdk.client.ErrorResponseException.class)
+    public void testCreate_OneReviewPerCustomerProductConstraint() throws Exception {
```

Comment:
The SDK is not responsible for processing the reply from the server? if so, how do you know that is working correctly if you don't test what is on the object?
Unless you mean that the creation and update times are not relevant information for the client and if so, why is it being provided?